packages/dependency-profiles/src/index.js

```diff
@@ -0,0 +1,298 @@
+      "name": "@fluentui-react-native/tester-win32",
+      "version": "0.27.18"
+    },
+    "@fluentui-react-native/codemods": {
+      "name": "@fluentui-react-native/codemods",
+      "version": "0.2.2"
+    },
+    "@fluentui-react-native/avatar": {
+      "name": "@fluentui-react-native/avatar",
+      "version": "1.2.23"
+    },
+    "@fluentui-react-native/badge": {
+      "name": "@fluentui-react-native/badge",
+      "version": "0.1.45"
+    },
+    "@fluentui-react-native/button": {
+      "name": "@fluentui-react-native/button",
+      "version": "0.26.0"
+    },
+    "@fluentui-react-native/callout": {
+      "name": "@fluentui-react-native/callout",
+      "version": "0.21.24"
+    },
+    "@fluentui-react-native/checkbox": {
+      "name": "@fluentui-react-native/checkbox",
+      "version": "0.16.28"
+    },
+    "@fluentui-react-native/contextual-menu": {
+      "name": "@fluentui-react-native/contextual-menu",
+      "version": "0.18.26"
+    },
+    "@fluentui-react-native/focus-trap-zone": {
+      "name": "@fluentui-react-native/focus-trap-zone",
+      "version": "0.9.23"
+    },
+    "@fluentui-react-native/focus-zone": {
+      "name": "@fluentui-react-native/focus-zone",
+      "version": "0.10.24"
+    },
+    "@fluentui-react-native/link": {
+      "name": "@fluentui-react-native/link",
+      "version": "0.16.22"
+    },
+    "@fluentui-react-native/menu-button": {
+      "name": "@fluentui-react-native/menu-button",
+      "version": "0.8.47"
+    },
+    "@fluentui-react-native/menu": {
+      "name": "@fluentui-react-native/menu",
+      "version": "1.0.53"
+    },
+    "@fluentui-react-native/notification": {
+      "name": "@fluentui-react-native/notification",
+      "version": "0.17.25"
+    },
+    "@fluentui-react-native/persona-coin": {
+      "name": "@fluentui-react-native/persona-coin",
+      "version": "0.11.13"
+    },
+    "@fluentui-react-native/persona": {
+      "name": "@fluentui-react-native/persona",
+      "version": "0.13.13"
+    },
+    "@fluentui-react-native/pressable": {
+      "name": "@fluentui-react-native/pressable",
+      "version": "0.9.22"
+    },
+    "@fluentui-react-native/radio-group": {
+      "name": "@fluentui-react-native/radio-group",
+      "version": "0.15.26"
+    },
+    "@fluentui-react-native/separator": {
+      "name": "@fluentui-react-native/separator",
+      "version": "0.12.12"
+    },
+    "@fluentui-react-native/stack": {
+      "name": "@fluentui-react-native/stack",
+      "version": "0.7.16"
+    },
+    "@fluentui-react-native/tabs": {
+      "name": "@fluentui-react-native/tabs",
+      "version": "0.7.25"
+    },
+    "@fluentui-react-native/text": {
+      "name": "@fluentui-react-native/text",
+      "version": "0.15.8"
+    },
+    "@uifabricshared/foundation-composable": {
+      "name": "@uifabricshared/foundation-composable",
+      "version": "0.11.1"
+    },
+    "@uifabricshared/foundation-compose": {
```

Comment:
note for future versions, are we able to / should we mark the @uifabricshared packages as deprecated, or not include them entirely?

Reply:
I want to say this is the goal of having all the V1 components. I believe Ruriko mentioned to me she wants to write a codemod to mark all the `@uifabricshared` stuff as deprecated in the future. Main holdout still using the old frameworks in my head is Callout..